packages/account/src/Containers/proof-of-ownership/expanded-card.tsx

```diff
@@ -1,116 +1,118 @@
+        payment_method_identifier: string,
+        identifier_type: string,
+        item_id: number,
+        documents_required: number
+    ) => {
         handleIdentifierChange(
-            name,
             formatIdentifier(payment_method_identifier, identifier_type),
             item_id,
-            item_index,
             documents_required
         );
     };
-    const handleIdentifierChange = (name, payment_method_identifier, item_id, item_index, documents_required) => {
-        setFieldValue(`${name}`, {
-            ...values.data?.[index]?.[item_index],
+    const handleIdentifierChange = (payment_method_identifier: string, item_id: number, documents_required: number) => {
+        setFieldValue(payment_method, {
+            ...values?.[payment_method],
             documents_required,
             id: item_id,
             payment_method_identifier,
-            is_generic_pm: card_details.is_generic_pm,
-            identifier_type: card_details.identifier_type,
+            is_generic_pm: card_details?.is_generic_pm,
+            identifier_type: card_details?.identifier_type,
         });
     };
-    const exampleLink = () =>
-        card_details?.identifier_type === IDENTIFIER_TYPES.card_number && (
-            <span
-                className='proof-of-ownership__card-open-desc-link'
-                key={0}
-                onClick={() => {
-                    setIsSampleModalOpen(true);
-                }}
-            >
-                {localize('See example')}
-            </span>
-        );
-    const formatIdentifier = (payment_method_identifier, identifier_type) => {
+
+    const formatIdentifier = (payment_method_identifier: string, identifier_type: string) => {
         let formatted_id = payment_method_identifier?.replace(/\s/g, '') || '';
-        if (identifier_type === IDENTIFIER_TYPES.card_number) {
-            if (
-                formatted_id.length !== 16 ||
-                (formatted_id.length === 16 && VALIDATIONS.has_invalid_characters(formatted_id))
-            ) {
+        if (identifier_type === IDENTIFIER_TYPES.CARD_NUMBER) {
+            if (formatted_id.length !== 16 || (formatted_id.length === 16 && hasInvalidCharacters(formatted_id))) {
                 return formatted_id;
             }
             formatted_id = `${formatted_id.substring(0, 6)}XXXXXX${formatted_id.substring(12)}`;
-        } else if ([IDENTIFIER_TYPES.email_address, IDENTIFIER_TYPES.user_id].some(s => s === identifier_type)) {
+        } else if ([IDENTIFIER_TYPES.EMAIL_ADDRESS, IDENTIFIER_TYPES.USER_ID].some(s => s === identifier_type)) {
             return formatted_id;
         }
         return formatted_id.replace(/(\w{4})/g, '$1 ').trim();
     };
-    const isSpecialPM = pm_icon => ['IcOnlineNaira', 'IcAstroPayLight', 'IcAstroPayDark'].some(ic => ic === pm_icon);
+
+    const isSpecialPM = (pm_icon: string) =>
+        ['IcOnlineNaira', 'IcAstroPayLight', 'IcAstroPayDark'].some(ic => ic === pm_icon);
```

Comment:
I know this is not part of your code but what is this for? we might need a better variable name.